sim/rogue/items_sets_pve.go

```diff
@@ -120,3 +120,53 @@ var ItemSetDarkmantleArmor = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+var ItemSetNightSlayerThrill = core.NewItemSet(core.ItemSet{
+	Name: "Nightslayer Thrill",
+	Bonuses: map[int32]core.ApplyEffect{
+		// Feint also grants Avoidance for 6 sec, reducing all damage taken from area of effect attacks from non-players by 50%
+		//2: Not yet implemented
+		//},
+		// Increases the critical strike damage bonus of your Poisons by 100%.
+		4: func(agent core.Agent) {
+			rogue := agent.GetCharacter()
+			rogue.OnSpellRegistered(func(spell *core.Spell) {
+				if spell.Flags.Matches(SpellFlagRoguePoison) {
+					spell.CritDamageBonus += 1
+				}
+			})
+		},
+		// Your finishing moves have a 5% chance per combo point to make your next ability cost no energy.
+		//https://www.wowhead.com/classic/spell=457342/clearcasting
+		6: 	func(agent core.Agent) {		
+			rogue := agent.(RogueAgent).GetRogue()			
+				
+			aura := rogue.RegisterAura(core.Aura{
+				Label:    "Clearcasting",
```

Comment:
Since clearcasting is a commonly used effect and auras can't have name conflicts, can you call this something like `"Clearcasting (S03 - Item - T1 - Rogue - Damage 6P Bonus)"`